src/applications/personalization/rated-disabilities/components/TotalRatedDisabilities.jsx

```diff
@@ -8,7 +8,13 @@ import {
 } from './TotalRatingStates';
 
 const TotalRatedDisabilities = props => {
-  const { loading, totalDisabilityRating, error } = props;
+  const {
+    loading,
+    totalDisabilityRating,
+    error: { status },
+  } = props;
+  const serverErrorRegex = /^5\d{2}$/;
+  const serviceErrorRegex = /^4\d{2}$/;
```

Comment:
cool. I've always done a check to see if the error code was >= 400 and <500. Stupid question, but if `status` is a Number, does the regex `test()` work?

Reply:
That's a good question. Right now, error codes for the other endpoint we use are coming back as strings. Currently, the endpoint for this value hasn't been built, so this could potentially change, but for now I'm assuming they will also come back as strings.

Reply:
I just checked and if you pass a number to `test()` it converts it to a string and does the comparison.